examples/99-advanced/reorder-edges.py

```diff
@@ -0,0 +1,97 @@
+"""
+.. _reorder_edges_example:
+
+Reorder Edges
+~~~~~~~~~~~~~
+
+This example demonstrates how to reorder edges based on connectivity.
+
+Boundary edges are extracted from a :class:`pyvista.PolyData` are unordered. We
+can use a simple mapping array based on :attr:`lines <pyvista.PolyData.lines>`
+to reorder the edges to make them sequential.
```

Comment:
We should make it clear that this only works if you have a single, closed boundary edge. For surfaces with holes inside you'll get multiple boundary edges and you'd have to call `split_bodies()` or `connectivity()` or similar first in order to get connected components, then running this for a given closed loop.